server/consumer.go

```diff
@@ -384,7 +387,8 @@ func checkConsumerCfg(config *ConsumerConfig, lim *JSLimitOpts, cfg *StreamConfi
 			return NewJSConsumerMaxRequestExpiresToSmallError()
 		}
 	}
-	if lim.MaxAckPending > 0 && config.MaxAckPending > lim.MaxAckPending {
+	if lim.MaxAckPending > 0 && config.MaxAckPending > lim.MaxAckPending ||
```

Comment:
Is it that `lim` are server limits and `limit` are account limits? If that's the case, I wonder if `srvLim` and `accLim` (or something similar) would make it easier to understand.

Reply:
yes, fixed